src/components/registration/steps.jsx

```diff
@@ -226,6 +226,7 @@ module.exports = {
     ChoosePasswordStep: intl.injectIntl(React.createClass({
         getDefaultProps: function () {
             return {
+                studentUsername: null,
```

Comment:
I don't think there's anything student-specific in here and we may use this in other non-student related contexts later - maybe `username` or `notEquals`?